.github/workflows/client.yml

```diff
@@ -188,4 +188,4 @@ jobs:
         with:
           go-version-file: "go.mod"
       - name: gofmt
-        run: gofmt -d -e .
+        run: if [ "$(gofmt -l . | wc -l)" -gt 0 ]; then exit 1; fi
```

Comment:
It works, but there is no context outputted in case of an error, see example execution: https://github.com/keep-network/keep-core/runs/8265007828?check_suite_focus=true

<img width="528" alt="image" src="https://user-images.githubusercontent.com/10741774/189292620-686240da-5b49-4dc9-b6bc-0e938785abc7.png">

I'd expect to see a list of the files that are failing the check.

Reply:
Yes. I thought it would be even better to be more precise and show the diff of what went wrong instead of just list.
I added it In https://github.com/keep-network/keep-core/pull/3252/commits/f24958dd9c3162a9898346f3097897fc1b1bfdf4.

Reply:
Awesome! https://github.com/keep-network/keep-core/runs/8302348675?check_suite_focus=true